data/src/test/java/com/teamagam/gimelgimel/data/message/repository/cache/room/dao/OutGoingMessagesDaoTest.java

```diff
@@ -58,28 +60,27 @@ public void deleteTopMessage() {
     mDao.insertMessage(outGoingChatMessage3);
     mDao.deleteTopMessage();
     mDao.deleteTopMessage();
+    mDao.deleteTopMessage();
 
     //Assert
-    assertTrue("tudu bom", mDao.getMessagesByOutGoingId().get(0).outGoingMessageId == 3);
+    assertThat(mDao.getMessagesByOutGoingId(), hasSize(0));
   }
 
-
   @Test
-  public void deleteAllMessages() {
+  public void getLastMessageTest() {
     //Arrange
     OutGoingChatMessageEntity outGoingChatMessage = mock(OutGoingChatMessageEntity.class);
-    OutGoingChatMessageEntity outGoingChatMessage2 = mock(OutGoingChatMessageEntity.class);
-    OutGoingChatMessageEntity outGoingChatMessage3 = mock(OutGoingChatMessageEntity.class);
 
     //Act
     mDao.insertMessage(outGoingChatMessage);
-    mDao.insertMessage(outGoingChatMessage2);
-    mDao.insertMessage(outGoingChatMessage3);
-    mDao.deleteTopMessage();
+    mDao.insertMessage(outGoingChatMessage);
+    mDao.insertMessage(outGoingChatMessage);
     mDao.deleteTopMessage();
     mDao.deleteTopMessage();
+    mDao.insertMessage(outGoingChatMessage);
 
     //Assert
-    assertTrue("tudu bom", mDao.getMessagesByOutGoingId().size() == 0);
+    assertTrue("Error: id is: " + mDao.getTopMessage().outGoingMessageId,
```

Comment:
Use the proper assertion tools.
`assertThat(mDao.geTopMessage().outGoingMessageId, is(4))`